src/Css.elm

```diff
@@ -6448,6 +6465,685 @@ outset =
 
 
 
+{- BORDER RADIUS -}
+
+
+{-| Sets [`border-radius`](https://css-tricks.com/almanac/properties/b/border-radius/) property.
+
+    borderRadius  (em 4)
+    borderRadius2 (em 4) (px 2)
+    borderRadius3 (em 4) (px 2) (pct 5)
+    borderRadius4 (em 4) (px 2) (pct 5) (px 3)
+
+**Note:** An implementation of shorthand property with supporting of elipse shape looks ugly,
+so if you wanna make somehting like this `border-radius: 4px 3px 6px / 2px 4px` by `elm-css`
+you should create a batch of `Style`:
```

Comment:
I didn't know how to say about the implementation problem more polite.. and also my English is not so cool as should be and I need your help as a native speaker 😅

Reply:
Wow, I never knew about `/` in `border-radius`!

I wonder if we should consider supporting this. We could do that by adding something like `, ellipticalRadius : Supported` to each of the `Value` arguments in `borderRadius`, then introducing this:
```
ellipticalRadius :
    Value { px : Supported, em : Supported, ... }
    -> Value { px : Supported, em : Supported, ... }
    -> Value { provides | ellipticalRadius : Supported }
```

so then you could call:

```
borderRadius (ellipticalRadius (px 10) (px 30))
```

thoughts @owanturist?

Reply:
The main problem is specific scheme of this property. I'd like to see the scheme like this:

```css
border-radius: 1px/2px 3px 4px/5px 6px/7px;
```

Where `1px`, `3px`, `4px` and `6px` are general values (like in `border-radius: 1px 3px 4px 6px`) and rest (`2px`, `5px` and `7px`) are addition values.

But it's a wrong signature and correct equivalent of example is:

```css
border-radius: 1px 3px 4px 6px / 2px 0 5px 7px;
```

So, you can see that supporting of `/` in `borderRadius` is not trivial, because all of them are valid:

```css
border-radius: 1px / 2px;
border-radius: 1px / 2px 3px;
border-radius: 1px / 2px 3px 4px;
border-radius: 1px / 2px 3px 4px 5px;

border-radius: 1px 2px / 3px;
border-radius: 1px 2px / 3px 4px;
border-radius: 1px 2px / 3px 4px 5px;
border-radius: 1px 2px / 3px 4px 5px 6px;

border-radius: 1px 2px 3px / 4px;
border-radius: 1px 2px 3px / 4px 5px;
border-radius: 1px 2px 3px / 4px 5px 6px;
border-radius: 1px 2px 3px / 4px 5px 6px 7px;

/* end so on... */
```

Well, if we wanna implement `ellipticalCorner` (which can be reused in `font` property if the rule would looks like `border-radius: 1px/2px 3px 4px/5px 6px/7px`) we could implement `borderRadius` from single argument to nine (`borderRadius9 (px 1) (px 2) (px 3) (px 4) ellipticalCorner (px 5) (px 6) (px 7) (px 8)` is most long) or pass tuple of list to `ellipticalCorner` but in this case we loose type checking. But even in `borderRadius9` we can't guarantee that developer pass `ellipticalCorner` in right place (`borderRadius9 (px 1) ellipticalCorner (px 2) (px 3) (px 4) (px 5) (px 6) (px 7) (px 8)` for example) so we should implement the property `borderRadius4elliptical4` for correct type checking:

```elm
borderRadius4elliptical4 (px 1) (px 2) (px 3) (px 4) (px 5) (px 6) (px 7) (px 8)
```

@rtfeldman sorry for this long story, but yesterday I broke my head while I was thinking about it 🤯